core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -15,6 +15,11 @@ class BattleDamageModifier(val vs:String,val modificationAmount:Float){
     fun getText(): String = "vs $vs"
 }
 
+enum class CombatantRole{
```

Comment:
TBH I don't really like this, using n enum where a boolean true/false would suffice, but OK I guess